docs/sql/queries.txt

```diff
@@ -89,6 +90,89 @@ A table can be aliased for the sake of brevity too::
     +-------------------+
     SELECT 1 row in set (... sec)
 
+Joins
+.....
+
+A Cross-join returns the cartesian product of the joined tables.
+The cartesian product contains every possible combinations of rows from
+the given tables ( T1 * T2 * ...)::
+
+    cr> select articles.name as article, colors.name as color
+    ... from articles cross join colors order by price limit 6;
```

Comment:
maybe put order by clause on separate line to make the from clause more visible, same for the 2 queries below.